packages/core/strapi/lib/types/core/common/service.d.ts

```diff
@@ -0,0 +1 @@
+export type Service = Record<keyof any, unknown>;
```

Comment:
What is this for?

Reply:
That's the default abstract type of a service, it's used to type registries (the Services one)